_test/test_transformation/test_cut_parameters.m

```diff
@@ -39,7 +39,9 @@ function test_double_multicut_expanded3d(obj)
             [proj, pbin, opt]= cut_inputs_tester(sqw_test,true,...
                 ortho_proj([1,0,0],[0,1,0]),...
                 [-1,1],[-1,0.5,1,1],[-1,0.5,1,1],[-1,0.8,0,1]);
-            op = ortho_proj([1,0,0],[0,1,0]);
+            op = ortho_proj([1,0,0],[0,1,0], ...
+                'alatt',sqw_test.data.proj.alatt, ...
+                'angdeg',sqw_test.data.proj.angdeg);
```

Comment:
unfortunately we need to duplicate `alatt` and `angdeg` according to the current design. Partially due to historical reasons. Partially because Tobyfit is not fully rewritten through generic projections. Probably sample should be single source of lattice (there are currently three here) but technically sample's lattice defines Tobyfit transformations and projection defines image/pix transformaton (should be  one but not implemented yet)